README.md

```diff
@@ -1,96 +1,127 @@
+## Step 3: Define database and schema variables
+By default, this package runs using your target database and the `zendesk` schema. If this is not where your Zendesk data is (for example, if your zendesk schema is named `zendesk_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    zendesk_schema: your_schema_name
-    zendesk_database: your_database_name
+    zendesk_database: your_destination_name
+    zendesk_schema: your_schema_name 
+```
+## Step 4: Disable models for non-existent sources
+This package takes into consideration that not every Zendesk account utilizes the `schedule`, `domain_name`, `user_tag`, `organization_tag`, or `ticket_form_history` features, and allows you to disable the corresponding functionality. By default, all variables' values are assumed to be `true`. Add variables for only the tables you want to disable:
+```yml
+vars:
+    using_schedules:            False         #Disable if you are not using schedules
+    using_domain_names:         False         #Disable if you are not using domain names
+    using_user_tags:            False         #Disable if you are not using user tags
+    using_ticket_form_history:  False         #Disable if you are not using ticket form history
+    using_organization_tags:    False         #Disable if you are not using organization tags
 ```
 
-### Adding Passthrough Columns
+## Step 5: Adding passthrough columns
 This package includes all source columns defined in the staging models. However, the `stg_zendesk__ticket` model allows for additional columns to be added using a pass-through column variable. This is extremely useful if you'd like to include custom fields to the package.
 ```yml
-# dbt_project.yml
-
-...
 vars:
   zendesk__ticket_passthrough_columns: [account_custom_field_1, account_custom_field_2]
 ```
-### Changing the Build Schema
-By default this package will build the Zendesk Support staging models within a schema titled (<target_schema> + `_zendesk_staging`) in your target database. If this is not where you would like you Zendesk Support staging data to be written to, add the following configuration to your `dbt_project.yml` file:
 
-```yml
-# dbt_project.yml
+## (Optional) Step 5: Additional configurations
+<details><summary>Expand to view configurations</summary>
+    
+### Change the build schema
+By default, this package builds the zendesk staging models within a schema titled (`<target_schema>` + `_stg_zendesk`) in your target database. If this is not where you would like your Zendesk staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
I feel we should use this time to standardize the custom schema to be in line with our other packages. So we would make the default schema to be `_zendesk_source` instead of following the `stg` naming convention.

We should then include this as a breaking change in the CHANGELOG.

Reply:
changed